frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/MainNavbarView.tsx

```diff
@@ -275,5 +298,93 @@ function CollectionSectionHeading({
     </SidebarHeadingWrapper>
   );
 }
-// eslint-disable-next-line import/no-default-export -- deprecated usage
-export default MainNavbarView;
+
+function SidebarOnboardingSection({
+  initialState,
+  isSidebarOpen,
+}: {
+  initialState: boolean;
+  isSidebarOpen: boolean;
```

Comment:
It's used for the width and the height of a fixed element
https://github.com/metabase/metabase/pull/48202/files#diff-9a9e635efbadeb805419e19d5623ee1e5d37ced0008f98996aa576aa69301137R316-R317

Reply:
but since the element goes away entirely when the sidebar is closed anyway, we don't need to conditionally set those. 👍

Reply:
But the sidebar is not removed from the DOM. IT would've been easier if it was.
We're just making it 0px wide.

This new element being `fixed` means that it is removed from the document flow. It doesn't know about its parent so I have to keep them in sync.

Take a look at what's happening with the sidebar's CSS on toggle.
![Kapture 2024-10-03 at 09 34 03](https://github.com/user-attachments/assets/374b73b2-f33b-43aa-bc9c-f0308fe60291)

Reply:
I changed the position to `sticky` which solves this (it inherits parent's width).